features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/state/ViewUIPerformanceReport.kt

```diff
@@ -37,15 +37,20 @@ internal data class ViewUIPerformanceReport(
 
     fun slowFramesRate(viewEndedTimeStamp: Long): Double = when {
         viewEndedTimeStamp - viewStartedTimeStamp <= minViewLifetimeThresholdNs -> 0.0
-        totalFramesDurationNs > 0.0 -> slowFramesDurationNs.toDouble() / totalFramesDurationNs
+        totalFramesDurationNs > 0.0 -> slowFramesDurationNs.toDouble() / totalFramesDurationNs * MILLISECONDS_IN_SECOND
         else -> 0.0
     }
 
     fun freezeFramesRate(viewEndedTimeStamp: Long): Double = when {
         viewEndedTimeStamp - viewStartedTimeStamp <= minViewLifetimeThresholdNs -> 0.0
         else -> max(
             0.0,
-            anrDurationNs.toDouble() / (viewEndedTimeStamp - viewStartedTimeStamp)
+            freezeFramesDuration.toDouble() / (viewEndedTimeStamp - viewStartedTimeStamp) * SECONDS_IN_HOUR
```

Comment:
question: why do we need to multiply values by `MILLISECONDS_IN_SECOND` and `SECONDS_IN_HOUR` accordingly?

Reply:
Initial implementation contained an error in scale. 
according to [schema](https://github.com/DataDog/rum-events-format/blob/45a80c1390b8ec886534f5f1b43763a6d9d0a643/schemas/rum/view-schema.json#L377)  
freeze_rate should be in in **seconds per hour**, slow_frames_rate in in **milliseconds per second**